doc/man7/RAND.pod

```diff
@@ -29,11 +29,14 @@ just use L<RAND_bytes(3)> to obtain random data.
 Having said that, there is one important rule to obey: Always check the error
 return value of L<RAND_bytes(3)> and don't take randomness for granted.
 
-For long-term secrets, you can use L<RAND_priv_bytes(3)> instead.
+For values that should remain secret, you can use L<RAND_priv_bytes(3)>
+instead.
 This method does not provide 'better' randomness, it uses the same type of CSPRNG.
-The intention behind using a dedicated CSPRNG exclusively for long-term secrets is
-that none of its output should be visible to an attacker (e.g used as salt value),
-in order to reveal as little information as possible about its internal state.
+The intention behind using a dedicated CSPRNG exclusively for private
+values is that none of its output should be visible to an attacker (e.g
+used as salt value), in order to reveal as little information as
```

Comment:
Some grammar pedants will insist upon offsetting things like "i.e." and "e.g." with commas (or parentheses) both before and after.

Reply:
"e.g." was missing the trailing point anyway, so I went ahead and also applied the "avoid contractions" suggestion.

Thanks @kaduk for spotting it!